docs/README.md

```diff
@@ -1,106 +1,109 @@
 
 [![Ubuntu Build](https://github.com/OpenVisualCloud/Media-Communications-Mesh/actions/workflows/ubuntu-build.yml/badge.svg)](https://github.com/OpenVisualCloud/Media-Communications-Mesh/actions/workflows/ubuntu-build.yml)
 [![Coverity status](https://scan.coverity.com/projects/30272/badge.svg?flat=1)](https://scan.coverity.com/projects/media-communications-mesh)
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/OpenVisualCloud/Media-Communications-Mesh/badge)](https://securityscorecards.dev/viewer/?uri=github.com/OpenVisualCloud/Media-Communications-Mesh)
 [![BSD 3-Clause][license-img]][license]
 
-## Table of Contents
-- [ Media Communications Mesh](#-media-communications-mesh)
-  - [Table of Contents](#table-of-contents)
-  - [Introduction](#introduction)
-  - [Media Proxy](#media-proxy)
-    - [Key Features](#key-features)
-    - [MemIF](#memif)
-  - [MCM SDK](#mcm-sdk)
-  - [Getting Started](#getting-started)
-    - [Prerequisites](#prerequisites)
-    - [Basic Installation](#basic-installation)
-  - [Usage](#usage)
-  - [Known Issues](#known-issues)
-  - [Support](#support)
-  - [Note](#note)
+![Media-Communication-Mesh](./img/media-communications-mesh-proto-1.webp)
 
 ## Introduction
 
 The Media Communications Mesh (MCM) enables efficient, low-latency media transport for media microservices for Edge, Edge-to-Cloud, and both private and public Cloud environments. The framework creates a secure, standards-based media data plane for inter-microservices communications using a new media proxy leveraging the [Media Transport Library (MTL)](https://github.com/OpenVisualCloud/Media-Transport-Library) and adds the necessary microservices control-plane communications infrastructure to implement any media control protocol.
 
-## Media Proxy
-
-The Media Proxy component in this project is responsible for transporting audio and video stream media data. It handles the routing and forwarding of media data between mesh nodes, ensuring low latency and efficient resource utilization.
-
-It serves as a Data Plane component in a media application's Service Mesh. Media Proxy provides Shared Memory APIs to abstract the complexity of media transport.
-
-More information about the Media Proxy component can be found in the [media-proxy](media-proxy) subdirectory.
-
-### Key Features
 
-- Enables zero memory copy and ultra-low-latency media transfers between containers.
-- Supports transport of video streams via compressed (JPEG XS) or RAW (uncompressed) protocols.
-- Offers compatibility with multiple media transport protocols (SMPTE ST 2110, RTSP, ...).
+## 1. Media Communications Mesh:
+•	Framework Introduction (MCM): is a framework designed for low-latency media transport. It caters specifically to Edge, Edge-to-Cloud, and Cloud environments, connecting media microservices.
```

Comment:
When I view .md document, those bullet points are not spaced out, they are squashed to a one line. The same issue can be observed in other places in this document.

Reply:
fixed